frontend/src/employee-frontend/components/Header.tsx

```diff
@@ -263,6 +263,17 @@ const Header = React.memo(function Header({ location }: RouteComponentProps) {
                     </Link>
                   </RequireRole>
                 ) : null}
+                {featureFlags.vasu ? (
```

Comment:
`featureFlags.vasu && <..` would be more idiomatic, but I see that ternary has been used in this file already.